CondTools/RunInfo/interface/LHCInfoCombined.h

```diff
@@ -37,13 +38,16 @@ class LHCInfoCombined {
       const edm::ESGetToken<LHCInfoPerLS, LHCInfoPerLSRcd>& tokenInfoPerLS,
       const edm::ESGetToken<LHCInfoPerFill, LHCInfoPerFillRcd>& tokenInfoPerFill,
       const edm::ESGetToken<LHCInfo, LHCInfoRcd>& tokenInfo,
-      bool useNewLHCInfo);
+      bool useNewLHCInfo,
+      bool isRealData);
 
   void setFromLHCInfo(const LHCInfo& lhcInfo);
   void setFromPerLS(const LHCInfoPerLS& infoPerLS);
   void setFromPerFill(const LHCInfoPerFill& infoPerFill);
 
-  float crossingAngle();
+  static inline bool useNewLHCInfoForDirectSimu = false;
```

Comment:
Do I understand correctly, this is a temporary way to switch between one mechanism and the other, right?
If so, it should be removed before the final PR. Also, I believe this should be const 😄

Reply:
It should be `const` 
If `isRealData()` will make it to the final PR this bool should be here unless we're 100% sure `Direct Simulation` will never be using new `LHCInfoPer*`